homeassistant/components/ambient_network/config_flow.py

```diff
@@ -0,0 +1,152 @@
+"""Config flow for the Ambient Weather Network integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from aioambient import OpenAPI
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import (
+    CONF_LATITUDE,
+    CONF_LOCATION,
+    CONF_LONGITUDE,
+    CONF_MAC,
+    CONF_RADIUS,
+    UnitOfLength,
+)
+from homeassistant.helpers.selector import (
+    LocationSelector,
+    LocationSelectorConfig,
+    SelectOptionDict,
+    SelectSelector,
+    SelectSelectorConfig,
+)
+from homeassistant.util.unit_conversion import DistanceConverter
+
+from .const import API_STATION_INDOOR, API_STATION_INFO, API_STATION_MAC_ADDRESS, DOMAIN
+from .helper import get_station_name
+
+CONF_USER = "user"
+CONF_STATION = "station"
+
+# One mile
+CONF_RADIUS_DEFAULT = 1609.34
+
+
+class AmbientNetworkConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle the config flow for the Ambient Weather Network integration."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Construct the config flow."""
+
```

Comment:
Please remove all these blank lines at the top of body in the functions. They are not needed and in many cases confusing.